addons/point_of_sale/static/src/app/screens/receipt_screen/receipt/order_receipt.xml

```diff
@@ -70,7 +70,8 @@
                 <span>Total</span>
                 <t t-foreach="props.data.tax_details" t-as="tax" t-key="tax.tax.id">
                     <span t-esc="tax.tax.letter || ''"/>
-                    <span><t t-esc="tax.tax.amount"/> %</span>
+                    <span t-if="tax.tax.amount_type != 'fixed'"><t t-esc="tax.tax.amount"/> %</span>
```

Comment:
`Why do we display the tax amount when it's already displayed in the following column ` Isn't it normal to display both the percentage and the actual value ? 
![image](https://github.com/odoo/odoo/assets/118446179/24f39ac6-ed4f-41a2-84c3-ca5272a044b2)
`why do we name that column TAX% and another one Tax` you can change this

Reply:
`Wouldn't the tax name or invoice_label be better suited for the first column ?` perhaps we can allow showing the tax name too, but i would imagine that some names might be quite long and thus not be displayed properly. What do you think about this. There are localisations that do indeed require the tax name to be shown, so it might not be a bad idea to provide a mechanism for it

Reply:
@vlst-odoo do you have time to provide some feedback ?

Reply:
Hello @mobt-odoo would you shed some light for us ? :bulb:

Reply:
@vlst-odoo FYI, extract from Discord conversation
> STGU: Mais donc la PR est bien en l'état ?
> MOBT: Yes